docs/user/build-customization.rst

```diff
@@ -344,30 +344,53 @@ Override the build process
 
 .. warning::
 
-   This feature is in a *beta phase* and could suffer incompatible changes or even removed completely in the near feature.
-   It does not yet support some of the Read the Docs' features like the :term:`flyout menu`, and ads.
-   However, we do plan to support these features in the future.
-   Use this feature at your own risk.
-
+   This feature is in *beta* and could change without warning.
+   It does not yet support some of the Read the Docs' features like the :term:`flyout menu`.
+   We do our best to not break existing configurations,
+   but use this feature at your own risk.
 
 If your project requires full control of the build process,
 and :ref:`extending the build process <build-customization:extend the build process>` is not enough,
-all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.
+all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands`.
 
 As Read the Docs does not have control over the build process,
-you are responsible for running all the commands required to install requirements and build your project properly.
-Once the build process finishes, the contents of the ``_readthedocs/html/`` directory will be hosted.
+you are responsible for running all the commands required to install requirements and build your project.
+
+Where to put files
+~~~~~~~~~~~~~~~~~~
 
+It is your responsibility to generate HTML and other formats of your documentation using :ref:`config-file/v2:build.commands`.
+The contents of the ``_readthedocs/<format>/`` directory will be hosted as part of your documentation.
+
+Supported :ref:`formats <downloadable-documentation:accessing offline formats>` are published if they exist in the following directories:
+
+* ``_readthedocs/html/`` (required)
+* ``_readthedocs/htmlzip/``
+* ``_readthedocs/pdf/``
+* ``_readthedocs/epub/``
```

Comment:
This can be updated now to `$READTHEDOCS_OUTPUT/<format>` 🎉

Reply:
:sunglasses:

Reply:
I think we probably want to keep it explicit, and mention the variable as an option?

Reply:
My point is that this may change in the future and people shouldn't use these path at all. They should use the variables. Otherwise, it could break in the future.

Reply:
I guess you also mentioned new environment variables @humitos - something like `$READTHEDOCS_OUTPUT_DIR_HTML` and `$READTHEDOCS_OUTPUT_DIR_PDF` that would be nicer for people to use.

So let's wait until that work is done :+1: